src/views/EdgeApplicationsCacheSettings/FormFields/FormFieldsEdgeApplicationCacheSettings.vue

```diff
@@ -303,7 +312,7 @@
           showButtons
           v-model="cdnCacheSettingsMaximumTtl"
           id="cdnCacheSettingsMaximumTtl"
-          :min="60"
+          :min="cdnCacheSettingsMaximumTtlMinimumValue"
           :max="31536000"
```

Comment:
sugestão: colocar esse numero mágico em uma variável que o represente 

```suggestion
          :max="..."
```